docs/en/observability/apm-ui/troubleshooting.asciidoc

```diff
@@ -142,7 +144,7 @@ You can also use the agent's public API to manually set a name for the transacti
 === Fields are not searchable
 
 In Elasticsearch, index templates are used to define settings and mappings that determine how fields should be analyzed.
-The recommended index templates for APM are installed by {fleet} when the Elastic APM integration is installed.
+The recommended index templates for APM come from the {es} apm-data plugin.
```

Comment:
do we want to stress that the apm-data plugin is preinstalled, to avoid users trying to "download" such a plugin? This point applies to many other mentions of apm-data plugin.